### DISCUSSION TITLE
Acaba l'sprint 4 i comença l'sprint 5.

### DISCUSSION THREAD

**Discussion by FherStk** [ElPuig/IMS#15] [1. IMS: general]
Bon dia a tothom,

Perdoneu el retard (confirmo que serà el meu eslògan), però l'sprint 4 (temps per a revisar els models de dades i la funcionalitat de l'esquelet) va acabar el dia 31 i el dia 1 va començar l'sprint 5. Com ja sabeu, hem definit els sprints perquè tinguin una durada d'un mes i durant aquest cinquè sprint ens toca el següent:

- Posar els textos de totes les vistes en anglès.
- Esbrinar com funciona el sistema de traduccions d'Odoo i aplicar-lo a les vistes perquè apareguin en Català.
- **L'anterior només aplica a les dades mestres** (tenim algun model o alguna vista del sistema de control d'assistència dins l'esquelet, no l'he esborrat per agilitzar el desenvolupament).

Salut i gràcies a tots,
Fer

**Comment by jherna** [comment]
Hola, m'he mirat una mica el sistema de traduccions.

Hi ha disponible a la meva branca (jordi-dev) el directori i fitxer tipus de com funciona el sistema de traduccions.
Hi ha diferents formats de fitxer, el que he pujat és el **_.po_** però hi ha també un **_.xml_**. Aquest fitxer cal descarregar-lo, primer triant l'idioma en què vols el teu odoo i després anant a traduccions en mode debug i als settings, descarregar el fitxer amb el formar escollit.

He vist que ara mateix l'idioma base barreja anglès i català, caldria veure que fem amb això. Tot i que si tenim la traducció en anglès ben implementada i la de català també, crec la barreja desapareixerà.

**Reply by FherStk** [reply]
Moltes gràcies, Jordi. 
Perdona que no t'hagi contestat abans. Crec que un cop hàgim posat tots els literals en anglès, el problema que comentes dels idiomes barrejats desapareixerà. Ens hem posat com a objectiu acabar les traduccions aquest mes, per tant, farem bon ús de la teva proposta. 

Gràcies de nou!
